test/Numerics/DGMethods/advection_diffusion/direction_splitting_advection_diffusion.jl

```diff
@@ -174,29 +175,46 @@ let
     mpicomm = MPI.COMM_WORLD
     FT = Float64
     numlevels = 2
-    polynomialorder = 4
     base_num_elem = 4
 
+    # This test doesn't do any heavy computational work,
+    # but compiles a lot of model/discretization combinations.
+    # Compilation times on the GPU are longer, so we only run the
+    # variable-degree case on the GPU
```

Comment:
Did you mean 

```suggestion
    # Compilation times on the GPU are longer, so we only run the
    # variable-degree case on the CPU
```
by any chance?

Reply:
No, I intentionally am only testing 1 parameter combination on the GPU (I just chose the polyorder = (4, 2) case). The reason being that the test just takes way too long to run on the GPU.

Reply:
Ok, sorry for misunderstanding/misreading. 

Perhaps 

```suggestion
    # Compilation times on the GPU are longer, so we only run one
    # variable-degree case on the GPU
```
would read better? To specify that the GPU uses only one config, compared to the other